cat-launcher/src/pages/ToolsPage/components/ExportGameData.tsx

```
@@ -0,0 +1,84 @@
+        <h2 className="text-2xl font-bold">Export Game Data</h2>
+        <p className="text-muted-foreground">
+          Zip and save your game data (saves, configs, etc.) to a
+          location of your choice.
+        </p>
+      </div>
+
+      <div className="space-y-4 max-w-md">
+        <div className="space-y-2">
+          <label className="text-sm font-medium">
+            Select Game Variant
+          </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)